icu4c/source/common/restrace.cpp

```diff
@@ -97,14 +98,14 @@ void FileTracer::traceOpenResFile(const char* path, const char* name) {
 
     icu::CharString filePath;
     filePath.append(path, status);
-    filePath.append("/", status);
+    filePath.append('/', status);
     filePath.append(name, status);
     filePath.append(".res", status);
 
     UTRACE_DATA1(UTRACE_VERBOSE, "%s", filePath.data());
     UTRACE_EXIT_STATUS(status);
 }
 
-#endif // U_ENABLE_TRACING
-
 U_NAMESPACE_END
+
+#endif // U_ENABLE_TRACING
```

Comment:
Say... in this macro and in the configure option, does it only apply to data tracing? As opposed to collation/conversion/... tracing? If so, please insert "data" in this macro and in the configure option.

Reply:
`U_ENABLE_TRACING` is pre-existing.  It covers all types of tracing.